arrow-libs/core/arrow-core/api/arrow-core.api

```diff
@@ -867,7 +867,6 @@ public final class arrow/core/NonFatalOrThrowKt {
 
 public final class arrow/core/None : arrow/core/Option {
 	public static final field INSTANCE Larrow/core/None;
-	public fun isEmpty ()Z
```

Comment:
@franciscodr sorry for dragging this PR on so long 🙈
We need to revert the change we made to `abstract fun isEmpty` in order to not break binary compatibility. The improvements we made in `isSome`.

Reply:
No need to apologize, @nomisRev. Thanks for your thorough review of this PR 😄